bin/spice/pkg/cli/cmd/init.go

```diff
@@ -22,15 +21,9 @@ spice init trader
 	Args: cobra.MinimumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		podName := args[0]
-		podManifestFileName := fmt.Sprintf("%s.yaml", strings.ToLower(podName))
-
-		rtcontext := context.NewContext()
-		podsPath := rtcontext.PodsDir()
-		podManifestPath := filepath.Join(podsPath, podManifestFileName)
-		appRelativeManifestPath := rtcontext.GetSpiceAppRelativePath(podManifestPath)
-
-		if _, err := os.Stat(podManifestPath); !os.IsNotExist(err) {
-			cmd.Printf("Pod manifest already exists at %s. Replace (y/n)? \n", appRelativeManifestPath)
+		podPath := "./spicepod.yaml"
```

Comment:
It seems podManifestPath previously was based on podName and now it is hardcoded ("./spicepod.yaml). Please double check

Reply:
Approved 👍